features/step_definitions/sign_up_steps.rb

```diff
@@ -0,0 +1,19 @@
+Given(/^I visit the index page$/) do
+  visit root_path
+end
+
+Given(/^I click "([^"]*)"$/) do |arg1|
+  click_link_or_button arg1
+end
+
+Then(/^I fill in "([^"]*)" with "([^"]*)"$/) do |title, content|
+  fill_in title, with: content
+end
+
+Then(/^I click Create$/) do
+  click_link_or_button 'Create'
+end
```

Comment:
When you re-use the click-link step for this one, you can delete this step definition.